permits/templatetags/permit_progressbar.py

```diff
@@ -53,6 +54,39 @@ def reverse_permit_request_url(name):
         instance=permit_request, enable_required=True, disable_fields=True, data={}
     ) if permit_request else None
 
+    # No actors saved so far for this permit request
+
+    if models.PermitRequestActor.objects.filter(permit_request=permit_request).count() == 0:
+
+        initial_actors = models.PermitActorType.objects.filter(
+            works_type__in = services.get_permit_request_works_types(permit_request)
+        ).exclude(type__in=models.PermitRequestActor.objects.filter(
+            permit_request=permit_request).values_list('actor_type',flat=True)
+        ).distinct('type')
+        actor_completed = False
+        actor_errors = []
+        for initial_actor in initial_actors:
+            actor_errors.append(initial_actor)
+    else:
+
+        actor_initial_forms = []
+        for permit_request_actor in models.PermitRequestActor.objects.filter(permit_request=permit_request):
+
+            actor_initial_forms.append({
```

Comment:
Maybe tomorrow

Reply:
Done. To prevent user from getting crasy with litte red light showing number of errors, it shows only the number of contact forms to fill ;-), so 2 new methods in services.py